packages/fx-core/src/component/driver/file/updateEnv.ts

```diff
@@ -16,9 +16,9 @@ import { logMessageKeys } from "../aad/utility/constants";
 import { DriverContext } from "../interface/commonArgs";
 import { ExecutionResult, StepDriver } from "../interface/stepDriver";
 import { addStartAndEndTelemetry } from "../middleware/addStartAndEndTelemetry";
-import { InvalidParameterUserError } from "./error/invalidParameterUserError";
```

Comment:
Is it safe to remove the old error ts file?

Reply:
It is safe now